### DISCUSSION TITLE
Copilot activation failed

### DISCUSSION THREAD

**Discussion by chil-trek** [community/community#10963] [Copilot Conversations]
Hi, 
I installed Copilot extension on VS Code but I get the following error message. Not sure how to get past this. 

![image](https://user-images.githubusercontent.com/86759773/152408596-00cb03cf-c190-4ca7-ac17-b482d92b96c4.png)

**Comment by davecheney** [comment]
Hello, I'm sorry you are having difficulties. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by chil-trek** [reply]
Hi Dave,
My VPN is off. I am wondering if this is because it has been a while since I got the copilot invite. Would it be possible to regenerate the key?

**Reply by davecheney** [reply]
I have confirmed your invite is active. The error "unable to get local issuer cerificate" indicates that someone, possibly your employer, internet service provider, or government, is using SSL interception technology to eavesdrop on your HTTPS traffic.
